prov/efa/test/efa_unit_test_use_device_rdma.c

```diff
@@ -0,0 +1,118 @@
+#include "efa_unit_tests.h"
```

Comment:
Did you intend to put the tests in `efa_unit_test_info.c` instead?

`*_use_device_rdma.c` looks out of place(too specific).

Reply:
moved into test_info.c.  Thanks.